### ISSUE TITLE
Подложка

### ISSUE THREAD

**Issue by submgr** [maximnara/cordova-plugin-yandex-ads#16]
Извините, если не по теме. Суть в том, что под рекламой появляется подложка. Сама подложка куда больше самой рекламы, но это еще не страшно. Самое грустное то, что эта подложка не адаптируется под тему — в темной теме подложка все еще белая, что выглядит ужасно. 

Пытаюсь разобраться хотя бы с цветом подложки. Подскажите, это исходит от этого плагина или из самого SDK рекламы Яндекса?

Помимо рекламы Яндекса показываю рекламу Admob и там такой проблемы нет — там подложка по теме адаптируется в плане цвета.

Буду рад, если подскажете как можно исправить это🥹

![image](https://github.com/maximnara/cordova-plugin-yandex-ads/assets/18516370/a0442ded-b2f2-4fdc-8338-cd212e639459)

**Comment by Winamp9** [comment]
Добрый день. А вы используете яндекс эдс и эдмоб в одном проекте?

**Comment by submgr** [comment]
Да, по умолчанию инициализируем баннер и другие типы рекламы от Admob, если
происходит неудача, то обращаемся к YandexAds

Проблемы описанную выше уже решил на своей стороне, в коде плагина есть
установка Background color, добавил в цвет два нуля (00) в конец, чтобы он
был прозрачным и саму подложку добавляю уже в своем веб-приложении.

Спасибо!

**Comment by maximnara** [comment]
Здравствуйте @submgr! Скорее всего вы указываете размер баннера и в случае если подходящего размера баннера нет, то подложка все равно остается того размера что вы указали.
